open-machine-learning-jupyter-book/assignments/ml-fundamentals/ml-overview-iris.ipynb

```diff
@@ -36,7 +36,7 @@
   },
```

Comment:
<p>Put iris data into COS and get the link.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>